### ISSUE TITLE
blender GIS not showing models 

### ISSUE THREAD

**Issue by azkor** [domlysz/BlenderGIS#764]
# **Blender and OS versions**

i use blender 3.0.0

# **Describe the bug**

sometimes not all the buildings show only a small part of them 

# **How to Reproduce**

i just make the osm elevation for buildings with the option  get OSM and i select it with the elev objet option only a smart part of buildings gets modeled 

# **Error message**
there is not an error messages 
https://user-images.githubusercontent.com/108616130/222148226-f6fe10db-0853-4db6-88c6-8869f29177af.PNG

**Comment by atseewal** [comment]
Are these same buildings created in the open street map source? Open Street Map doesn't always have every building mapped.

**Comment by azkor** [comment]
yes They are on google earth i can preview it  in 3D but when i try it on blender GIS only a half of it gets a 3d model heres the location its a stadium 
https://earth.google.com/web/search/Stade+Hammadi+Agrebi,+Radès/@36.7495396,10.28598254,14.20614568a,516.21807538d,35y,85.30816465h,79.26960036t,-0r/data=CocBGl0SVwolMHgxMmZkNDllODVlZmRhNmJmOjB4NTk0MWQ3ODE5NmRjYTJhMBk9uDtrt19CQCHfc7vytIskQCocU3RhZGUgSGFtbWFkaSBBZ3JlYmksIFJhZMOocxgBIAEiJgokCSXpTyyOY0JAEQuIjE1AYUJAGRDUzDSPkCRAIXhojyAtfSRA

**Comment by atseewal** [comment]
OSM is completely separate from Google. Check https://www.openstreetmap.org/ instead.
